src/FSharp.NET.Sdk/FSharp.NET.Sdk.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
just add a `netstandard1.4` as target atm, or something like that. it doest matter